advanced_functionality/distributed_tensorflow_mask_rcnn/mask-rcnn-s3.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p>May be more helpful to move this cell up, or add similar cells immediately after those where we ran the <code>build_and_push</code> scripts. Less hopping around for the user.</p><p> </p><p>For example, after build_and_push for the TensorPack image:</p><p><code>tensorpack_image = #&lt;amazon-ecr-uri&gt;</code></p><p> </p><p>And after the AWS Samples image:</p><p><code>aws_samples_image = #&lt;amazon-ecr-uri&gt;</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
This is a good idea. I will do so and then use these variables in the cell below.

Reply:
Done